test/remote/gateways/remote_moneris_test.rb

```diff
@@ -156,13 +178,6 @@ def test_cvv_match_when_enabled
     assert_equal({'code' => 'M', 'message' => 'CVV matches'}, response.cvv_result)
   end
 
-  def test_cvv_no_match_when_enabled
```

Comment:
The test failed and I don't understand why it wouldn't be expected to fail. The test above is the exact same (other than the amount) and expects a success response... (same response the deleted test was actually getting)